static/sass/elements/chromedash-metrics.scss

```diff
@@ -18,6 +18,21 @@
       display: none;
     }
 
+    label {
+      font-weight: 600;
+      // word-wrap: break-word;
```

Comment:
Will do (I was just moving the existing rules around - thought the commented out stuff might still carry some significance).